.buildkite/metricbeat/pipeline.yml

```
@@ -327,6 +328,39 @@ steps:
           - github_commit_status:
               context: "metricbeat: Win 2022 Unit Tests"
 
+      - label: ":ubuntu: Metricbeat: FIPS ECH Integration Tests"
+        env:
+          ASDF_TERRAFORM_VERSION: "1.9.3"
+          ASDF_PYTHON_VERSION: "3.9.13" # Not needed by ECH tests, but needed by VM
+        command: |
+          .buildkite/scripts/custom_fips_ech_test.sh metricbeat
+        retry:
+          automatic:
+            - limit: 1
+        agents:
+          provider: "aws"
+          imagePrefix: "${IMAGE_UBUNTU_X86_64_FIPS}"
+          instanceType: "m5.xlarge"
+        artifact_paths:
+          - "metricbeat/build/*.xml"
+          - "metricbeat/build/*.json"
+          - "metricbeat/build/integration-tests/*"
+          - "metricbeat/build/integration-tests/Test*/*"
+          - "metricbeat/build/integration-tests/Test*/data/**/*"
+        plugins:
+          - test-collector#v1.10.2:
```

See https://github.com/buildkite-plugins/test-collector-buildkite-plugin/releases/tag/v1.11.0

```suggestion
          - test-collector#v1.11.0:
```